util/Version.h

```diff
@@ -1,15 +1,41 @@
 #ifndef _Version_h_
 #define _Version_h_
 
+/** @file
+ * @brief  Declares free functions to access the application and dependencies
+ *      versions.
+ */
+
 #include <string>
 #include <map>
 
 #include "Export.h"
 
+/** @brief  Returns the version string of FreeOrion.
+ *
+ * @return  The version string with the format
+ *      <em>VERSION BRANCH_NAME [build DATE.SHORT_COMMIT_HASH] BUILD_SYSTEM</em>
+ */
 FO_COMMON_API const std::string& FreeOrionVersionString();
 
+/** @brief  Returns a map of dependency versions.
+ *
+ * @return  A map with the dependency versions.  The key represents the
+ *      dependency name, the value corresponding dependency version string.
+ *
+ * @bug  The current implementation doesn't expose all dependencies of all
```

Comment:
I would consider it a bug, given the current indirect usage of this function by HumanClientApp::HumanClientApp() and ServerApp::ServerApp() and the in-code comment written by you regarding Vorbis and Ogg.